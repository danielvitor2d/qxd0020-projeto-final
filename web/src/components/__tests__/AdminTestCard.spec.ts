import { shallowMount } from '@vue/test-utils';
import { describe, expect, it } from 'vitest';
import AdminTestCardVue from '../AdminTestCard.vue';

describe('ComponentName', () => {
  it('renders correctly', () => {
    const test = {
      id: 1,
      title: 'Test Title',
      description: 'Test Description',
      numResponses: 5
    };

    const wrapper = shallowMount(AdminTestCardVue, {
      props: {
        test: test
      }
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
