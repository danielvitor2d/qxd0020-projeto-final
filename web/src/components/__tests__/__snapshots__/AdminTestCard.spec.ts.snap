// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ComponentName > renders correctly 1`] = `
"<div class=\\"bg-gray-100 rounded-lg shadow-lg p-6\\">
  <h2 class=\\"text-xl font-semibold text-gray-800 mb-4\\">Test Title</h2>
  <p class=\\"text-gray-600 mb-4\\">Test Description</p>
  <div class=\\"flex justify-between items-center\\"><span class=\\"text-sm text-gray-500\\">5 respostas</span>
    <div><button class=\\"bg-blue-500 hover:bg-blue-600 text-white py-2 px-4 rounded mr-2\\"> Ver detalhes </button><button class=\\"bg-red-500 hover:bg-red-600 text-white py-2 px-4 rounded\\"> Remover </button></div>
  </div>
</div>"
`;
